Models/Chat.cs

```diff
@@ -1,21 +1,28 @@
 ﻿using System;
 using System.Collections.Generic;
 
-namespace Tinker_Back.Models;
-
-public partial class Chat
+namespace Tinker_Back.Models
 {
-    public int Id { get; set; }
 
-    public string Name { get; set; } = null!;
+    public class Chat
+    {
+        public Chat(string name, string description)
+        {
+            Name = name;
+            Description = description;
+        }
+        public int Id { get; set; }
+
+        public string Name { get; set; } = null!;
 
-    public bool? IsGroupChat { get; set; }
+        public bool? IsGroupChat { get; set; }
 
-    public string? Description { get; set; }
+        public string? Description { get; set; }
 
-    public bool? IsPinned { get; set; }
+        public bool? IsPinned { get; set; }
```

Comment:
How are we going to manage if chat is pinned for single user, accorfing to this logic, chat will be pinned for all of them.
Suugestion is to move this logic into UserToChat, or how does it calls, so we will be able to manage if chat is pinned for concrete user